windows-driver-docs-pr/display/iddcx-versions.md

```diff
@@ -17,7 +17,8 @@ This page also lists the IddCx version (and **IddCxGetVersion** value) shipped w
 
 | IddCx version | IddCxGetVersion returned | Changes from previous public version |
 | ------------- | ------------------------ | ------------------------------------ |
-| [1.10](iddcx1.10-updates.md) | 0x1A00 | Add HDR10 and SDR Wide Color Gamut (WCG) support |
+| [1.10](iddcx1.10-updates.md) | 0x1A80 | Add HDR10 and SDR Wide Color Gamut (WCG) support - Windows 2024 |
```

Comment:
I see from your internal spec PR that HDR and runtime power management were added for 0x1A80. Add runtime power management here? Also, FYI that I'm seeking clarification on external references to Win 2024 for Ge ...